shared/src/main/scala/mlscript/JSBackend.scala

```diff
@@ -203,6 +204,7 @@ class JSBackend {
       JSArray(terms map { case (_, (term, _)) => translateTerm(term) })
     case Subs(arr, idx) =>
       JSMember(translateTerm(arr), translateTerm(idx))
+    case Assign(field, value) => ???  // TODO
```

Comment:
I think it should be easy to implement this case while we're at it. You just need to generate a JS assignment expression.

Reply:
I find that I also need to return a `unit`, but I don't know how @_@

Reply:
Okay, I think I've find a way